src/StorageSync/StorageSync/Az.StorageSync.psd1

```diff
@@ -119,7 +120,7 @@ PrivateData = @{
         # IconUri = ''
 
         # ReleaseNotes of this module
-        ReleaseNotes = '* Fix bug for not passing OFflineDataTransferProxy to SDK'
+        ReleaseNotes = '* Support for invoke change detection command'
```

Comment:
No need to edit this

Reply:
> No need to edit this

Should I just discard my change? Or edit it in some other way?

Reply:
I believe my change is still needed since new cmdlet name needs to be listed. Removed change to ReleaseNotes.